nurseconnect/tasks.py

```diff
@@ -0,0 +1,27 @@
+import requests
```

Comment:
I'm not sure, since I used an example from telkom mothership where they grabbed data from the google play store using the tasks.py file.